swarm-production/docker-stack.yml

```diff
@@ -89,6 +94,90 @@ services:
         delay: "10s"
         monitor: "5s"
 
+  queue:
+    image: 'wurstmeister/kafka:1.0.0'
+    environment:
+      KAFKA_ADVERTISED_PORT: 9092
+      KAFKA_ADVERTISED_HOST_NAME: queue
+      KAFKA_CREATE_TOPICS: 'backbeat-replication:1:1'
+      KAFKA_ZOOKEEPER_CONNECT: 'quorum:2181'
+    ports:
+      - 9092
+    networks:
+      - backend
+    volumes:
+      - "queue-journal:/kafka:rw"
+    depends_on:
+      - quorum
+    deploy:
+      mode: replicated
+      replicas: 1
+      placement:
+        constraints:
+          - node.labels.io.zenko.type == storage
+
+  quorum:
+    image: 'zookeeper:3.4.11'
+    ports:
+      - 2181
+    networks:
+      - backend
+    volumes:
+      - "quorum-data:/data:rw"
+      - "quorum-datalog:/datalog:rw"
+    deploy:
+      mode: replicated
+      replicas: 1
+      placement:
+        constraints:
+          - node.labels.io.zenko.type == storage
+
+  backbeat-producer:
+    image: 'zenko/backbeat:pensieve-2'
```

Comment:
for search we have a separate image that is just one backbeat producer for everything. we need to reconcile this.

Reply:
I thought search would not be swarm-based? This service just moves events from MD to kafka so I don't see why it would be incompatible.

Reply:
we have a different branch of backbeat for search which we were intending to go into Zenko CE but we can update it later.